consoles/mesos.html

```diff
@@ -0,0 +1,61 @@
+{{ template "head" . }}
+
+{{ template "prom_right_table_head" }}
+<tr>
+    <th colspan="2">Statistics</th>
+</tr>
+<tr>
+  <td>Mesos Jobs (active)</td>
+  <td>{{ template "prom_query_drilldown" (args "count(mesos_task_cpu_limit{job='mesos'})") }}</td>
+</tr>
+<tr>
+  <td>Memory (used / allocated)</td>
```

Comment:
I'm not sure this is meaningful in Mesos terms, as used is allocated. Allocated compared to the value in --resources would be the more appropriate metric, as that's how much of the slave's resources is reserved.